java/src/jmri/jmrix/openlcb/swing/downloader/LoaderPane.java

```diff
@@ -82,10 +82,10 @@ public void initComponents(CanSystemConnectionMemo memo) {
         p.setLayout(new WrapLayout());
         p.add(new JLabel("Address Space: "));
 
-        spaceField = new JTextField("" + 0xEF);
+        spaceField = new MemorySpaceSelector(0xEF);
         p.add(spaceField);
         selectorPane.add(p);
-        spaceField.setToolTipText("The decimal number of the address space, e.g. 239");
+        spaceField.setToolTipText("The number of the address space, e.g. 239 or 0XEF");
```

Comment:
Can the hex value be `0xEF`? If so, I suggest that you write the hex value that way.

`0xEF` is easier to see what it is, compared to `0XEF`. The X character blends in as a part of the hex number.